apps/web/src/app/[orgShortcode]/layout.tsx

```diff
@@ -29,28 +32,36 @@ export default function Layout({
     return (
       <div className="flex h-full w-full flex-col items-center justify-center gap-2">
         <SpinnerGap className="text-base-11 h-20 w-20 animate-spin" />
+        <span className="text-slate-11 font-bold">UnInbox is Loading...</span>
```

Comment:
we should always wrap `UnInbox` in a span with `font-display` so it shows our logo
can you try this and send a screenshot for confirmation please